packages/ember-glimmer/tests/integration/helpers/unbound-test.js

```diff
@@ -250,11 +250,17 @@ moduleFor('Helpers test: {{unbound}}', class extends RenderingTest {
       return a.join('');
     });
 
-    this.render(`{{unbound (repeat foo count=bar)}} {{repeat foo count=bar}} {{unbound (repeat foo count=2)}} {{repeat foo count=4}}`, {
-      foo: 'X',
-      numRepeatsBinding: 'bar',
```

Comment:
Good find, doesn't look like it. Also doesn't appear to me to be relevant to the test. Tests remain green after removal of that and the deprecation expectation.

I will amend my commit with these changes.